roles/openshift_rhacm/tasks/delete_observability.yml

```diff
@@ -18,9 +19,24 @@
       - pull-secret-secret.yaml
       - thanos-object-storage-secret.yaml
     kustomize_op: delete
+  when: rhacm_observability_use_existing_s3 == "false"
```

Comment:
Comparing boolean to string. Should probably read:
```
when: not rhacm_observability_use_existing_s3
```

The change is required in multiple places.